src/main/java/seedu/address/logic/commands/DeleteClientCommand.java

```diff
@@ -0,0 +1,45 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+
+/**
+ * eletes a client identified using it's displayed index from the address book.
+ */
+public class DeleteClientCommand extends Command {
+
+    public static final String COMMAND_WORD = "removeClient"; //TODO: Change back to delete
```

Comment:
removeClient might be too long a command

Reply:
yep! This will change back to delete once everyone migrates to client instead of person. Then there's no difference btwn 
`add` and `addClient`

For V1.2 my original intention (in case I can't get the product working by next week) is to just have a add/removeClient feature